free-programming-books-id.md

```diff
@@ -73,7 +73,7 @@
 ### MySQL
 
 * [Tutorial MySQL untuk Pemula Hingga Mahir](https://umardanny.com/tutorial-mysql-untuk-pemula-hingga-mahir-ebook-download-pdf/)
-
+* [3 Days With Mysql for your Application](https://play.google.com/store/books/details/Onesinus_Saut_Parulian_3_Days_With_Mysql_For_Your?id=MbdTDwAAQBAJ) - Onesinus Saut Parulian, Google Play Books (*account required*)
 
 ### Node.js
```

Comment:
Please refer to the comment made on the other [PR](https://github.com/EbookFoundation/free-programming-books/pull/4556#discussion_r509096714) you made for adding book MySQL for how to fix the issue here.